frontend/packages/console-shared/src/components/catalog/catalog-view/CatalogCategories.tsx

```diff
@@ -36,14 +38,23 @@ const CatalogCategories: React.FC<CatalogCategoriesProp> = ({
     return (
       <VerticalTabsTab
         key={id}
-        title={label}
         active={active}
         className={tabClasses}
-        onActivate={() => onSelectCategory(id)}
         hasActiveDescendant={hasActiveDescendant(selectedCategory, category)}
         shown={toplevelCategory}
         data-test={`tab ${id}`}
-        href={getURLWithParams(CatalogQueryParams.CATEGORY, id)}
```

Comment:
Should we open this as an upstream PF issue if the link isn't behaving properly?

cc @jcaianirh

Reply:
The problem is that all PF links uses just `a`nchors, I think there is no quick solution to replace or make them more customizable so that use react-router-dom `Link`s automatically, or?!

Reply:
@spadgett can this be resolved here, so we get this PR merged?

Reply:
@gruselhaus I have no objections. My comment has been addressed, thanks!